### DISCUSSION TITLE
possibility to use other llama model like llama3.2

### DISCUSSION THREAD

**Discussion by blakeandreasen-pdc** [mariocandela/beelzebub#139] [Ideas]
Currently, only Llama 3 works. Please add support for Llama 3.2 or other Llama models for more flexibility.

**Comment by mariocandela** [comment]
Hi @band8266,

Thanks for the suggestion, I will soon integrate LangChain as an integration layer towards the various LLM providers, to be agnostic and compatible with all LLM.

I will add you in the release notes so you don't miss the update :smile: 

Thanks for your time 

Chears 

Mario
